sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -297,9 +272,6 @@ document_translation_client = DocumentTranslationClient(endpoint, credential)
 jobs = document_translation_client.list_submitted_jobs()  # type: ItemPaged[JobStatusResult]
 
 for job in jobs:
-    if not job.has_completed:
```

Comment:
To make sure I understand, are you not finishing the job because Python can't rehydrate an operation?

Reply:
We can, but it's not pretty. Choosing not to showcase on the README as we don't show the `continuation_token` rehydration for other services either.